### PR TITLE
scriptconfig: add package

### PR DESCRIPTION
**Pull request by vicentebolea**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Erotemic** [comment]
Yes, I am ok being added a maintainer.

**Comment by vicentebolea** [comment]
@conda-forge/help-python, ready for review! CI passing.

File: recipes/scriptconfig/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - python
+    - setuptools >=41.0.1
+    - pip
+  run:
+    - python
+    - ubelt >=1.3.6
+    - pyyaml >=5.4.1  # [py<310 and py>=38]
+    - pyyaml >=6.0.0  # [py<312 and py>=310]
+    - pyyaml >=6.0.1  # [py>=312]
```
Comment (ocefpaf) [inline recipes/scriptconfig/meta.yaml:27]: See my comment in https://github.com/conda-forge/staged-recipes/pull/27277/files#r1741559350.
We can run the whole test suit to be sure but I doubt these different versions are really needed.

**Comment by vicentebolea** [comment]
@ocefpaf ready for review/merge; review comment addressed; CI passing

**Pull request merged**